experimenter/experimenter/experiments/tests/api/v5/test_queries.py

```diff
@@ -2138,6 +2225,102 @@ def test_query_feature_order(self):
         )
 
 
+class TestNimbusAllExperimentsMetaQuery(GraphQLTestCase):
+    GRAPHQL_URL = reverse("nimbus-api-graphql")
+
+    @property
+    def default_headers(self):
+        return {settings.OPENIDC_EMAIL_HEADER: "user@example.com"}
+
+    def test_query(self):
```

Comment:
You can probably get rid of this test altogether, the other one tests both the inclusion and exclusion cases?